README.md

```diff
@@ -50,6 +50,59 @@ You'll then be able to query this collection of documents (pages) from Typesense
 
 You can also add any arbitrary fields to the document, by adding the `data-typesense-field` data attribute to any HTML element.
 
+## Internationalization (i18n)
+
+Internationalized sites will often have a page structure which doesn't quite
+fit into specific directories. For example:
+
+```
+/die-cut-stickers/       => Product collection, locale = en-gb
+/de/gestanzte-sticker/   => Product collection, locale = de-de
+/de/uber-uns/kultur/     => Pages collection, locale = de-de
+/de/uber-uns/buro/       => Pages collection, locale = de-de
+```
+
+To support this add the following in `gatsby-config.js`
+
+```json
+  plugins: [
+    ...
+    {
+      resolve: `gatsby-plugin-typesense`,
+      options: {
+        rootDir: `${__dirname}/public`, // Required
+        includeTags: { "data-typesense-model": "product" },
```

Comment:
The word `includeTags` almost makes it sound like these are the only tags that will be indexed by the plugin. How about calling it something like `includePagesWithTags`. 

Also, I know the tag name here is just an example, but it makes it sound like it configures the Typesense Collection that the page is indexed to. How about we have the example as:

```
includePagesWithTags: {"data-typesense-include-page": "product"}
```